R/prepare-admin-panel.R

```diff
@@ -1,3 +1,10 @@
+# define function to get username
+get_user <- function(session) {
+  username <- shiny::isolate(shiny::parseQueryString(session$clientData$url_search)$username)
+  if (is.null(username)) username <- "unknownUser"
+  shiny::req(username)
+  return(username)
+}
```

Comment:
Convenience function to handle sessions without any user. We might remove this in the future.